values-schema.yaml

```diff
@@ -1462,16 +1470,9 @@ properties:
         properties:
           _rawValues:
             $ref: '#/definitions/rawValues'
-          enabled:
```

Comment:
Looks like property removal. If so please add it to the values-changes.yaml file.

Reply:
never used or produced by code so no need